deployments/kubernetes/chart/reloader/values.yaml

```diff
@@ -78,6 +78,7 @@ reloader:
     resources: {}
     pod:
       annotations: {}
+    priorityClassName: "foo"
```

Comment:
```suggestion
    # priorityClassName: "foo"
```

Please keep this commented out since we don't want this as a default value.

Reply:
Sorry must have missed this after testing.